fendermint/vm/genesis/src/lib.rs

```diff
@@ -47,6 +47,8 @@ pub struct Genesis {
     /// IPC related configuration, if enabled.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub ipc: Option<ipc::IpcParams>,
+    /// The application protocol version.
+    pub app_version: u64,
```

Comment:
Don't you think it would be okay to say that at genesis the app version is always 0, and subsequent upgrades can set it to whatever they like?

Reply:
The only reason I could think of for _not_ to start with 0 is if there was some kind of a configuration that operators can opt-into, which results in a descriptor that other chains can interpret, e.g. if the thing supported multiple protocols, it could describe that "this chain supports IPC v6.0" and then other chains that only do v5.x would know that they arent' compatible.  

But we have a single number, which I think like a private version number for this ledger, which just started. We think upgrades will be coming from either endorsed IPC upgrades mixed with subnet specific ones, so the sequence of upgrades will mean completely different things in different subnets. It's just a means to compare nodes on the same subnet.

Reply:
@aakoshh Are you suggesting removing `app_version` from the `Genesis` and just assume for now that is should always start with `0`? That makes sense, we _could_ add that in later if needed I guess.

Reply:
Done, removed from Genesis!